src/Validator.php

```diff
@@ -60,6 +60,21 @@ function ($value) {
     }
 
     /**
+     * Assert that each variable is a number.
+     *
+     * @return \Dotenv\Validator
+     */
+    public function isInteger()
+    {
+        return $this->assertCallback(
+            function ($value) {
+                return (is_int($value) == true);
```

Comment:
I am wondering if this will ever actually be true. ENV variables are always strings, so I don't think this will ever pass. What you need to do instead is to use something like `ctype_digit` or a regex pattern like `[0-9]+` to test the actual contents of the string instead of the PHP type.